src/Dashboard/Data/RecentInvocationEntry.cs

```diff
@@ -64,11 +65,23 @@ public string DisplayTitle
             get { return _succeeded; }
         }
 
+        // Summary heartbeat from fast logger
+        // This heartbeat is for the function-instance (not for the host).  
+        public DateTime? FunctionInstanceHeartbeatExpiry { get; set; }
+
         public HeartbeatDescriptor Heartbeat
         {
             get { return _heartbeat; }
         }
 
+        public static string FunctionInstanceHeartbeatExpiredTimeKey1
```

Comment:
nit: remove the extraneous "1" from the end of this property name? Actually you could just remove the entire property and make the field internal if all you're trying to do is make the value accessible.